src/bl/commentFacade.ts

```diff
@@ -0,0 +1,117 @@
+import { getRepository } from 'typeorm'
+
+import { NotFoundException } from './errors/NotFoundException'
+import { MissingParameterException } from './errors/MissingParameterException'
+import { Comment } from '../entities/comment'
+// import { ParamsExtractor } from './paramsExtractorv2'
+import { CardFacade } from './cardFacade'
+import { UserFacade } from './userFacade'
+
+import { RealTimeFacade } from './realtimeFacade'
+import { commentCreated, commentDeleted } from './realtime/realtimeComment'
+
+export class CommentFacade {
+
```

Comment:
I think now, you can delete this comment ;)

Reply:
?